jdaviz/configs/cubeviz/plugins/tools.py

```diff
@@ -3,10 +3,16 @@
 
 from glue.config import viewer_tool
 from glue.viewers.common.tool import CheckableTool
+from glue.core.link_helpers import LinkSame
+from specutils import Spectrum1D
 
 from jdaviz.core.events import SliceSelectWavelengthMessage, SliceToolStateMessage
+from jdaviz.core.marks import SelectedPixel
+from jdaviz.core.events import SnackbarMessage
+
+
+__all__ = ['SelectSlice', 'SpectrumPerSpaxel']
```

Comment:
I tried to rebase this PR locally so I can review it on top of latest changes and I noticed this. I purposely make this empty list in #1346 to have this be consistent with other configs.

```suggestion
__all__ = []
```

Reply:
Got it, I'll commit it then.